src/reanimated2/commonTypes.ts

```diff
@@ -13,11 +13,14 @@ export interface StyleProps extends ViewStyle, TextStyle {
  *
  * Shared values are defined using [useSharedValue](https://docs.swmansion.com/react-native-reanimated/docs/core/useSharedValue) hook. You access and modify shared values by their `.value` property.
  */
-export interface SharedValue<Value> {
+export interface SharedValue<Value = unknown> {
```

Comment:
Shouldn't we also update our docs (in particular "Type definitions" section [here](https://docs.swmansion.com/react-native-reanimated/docs/core/useSharedValue/))?

Reply:
Yes.

Reply:
Added in 233b65d